scripts/tools/GlobalTimestamps.yml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Tool Experimental Extension for Global Metrics Timestamps"
+version: "1.5"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Global Metrics Timestamps Experimental Extension Name"
+version: "1.5"
+name: $T_GLOBAL_METRICS_TIMESTAMPS_EXP_NAME
+value: '"$XT_experimental_global_metrics_timestamps"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Global Metrics Timestamps Experimental Extension Version(s)"
+version: "1.5"
+name: $x_global_metrics_timestamps_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Returns global device timestamps synchronized with metrics timesstamps, optionally mapped to host."
+class: $tMetricGroup
+name: GetTimestampsExp
+decl: static
+details:
+    - "The application may call this function from simultaneous threads."
+params:
+    - type: "$t_metric_group_handle_t"
+      name: hMetricGroup
+      desc: "[in] handle of the metric group"
+    - type: "$x_bool_t"
+      name: mappedToHost
```

Comment:
The original request was to sync with device, which I know how to satisfy. However, its unclear if synch with host can be done. Does L0 core have APIs to synch device and host? if so, its internal implementation could be used here.

Reply:
"mappedToHost" to  "SynchronizedWithHost" ?

Reply:
@matcabral : Core has something similar, but dont know if it could be exactly be used. This is an experimental extension, so if we are not suer, we could add the host support later, or remove it. Please let us know which you prefer.

Reply:
found that MDAPI also has support to synch with host. Therefore, I think makes more sense to have in on the initial implemenation .